### DISCUSSION TITLE
Is this a real Tokio site?

### DISCUSSION THREAD

**Discussion by tianyizheng02** [tokio-rs/tokio#5853] [General] [answered]
Hello,

I'm a maintainer of a GitHub repo for tracking tech internships, and someone just opened a PR adding what appears to be a Tokio internship position: pittcsc/Summer2024-Internships#1999. The linked webpage is on https://tokio.uk.ms/, not the official https://tokio.rs. Is this an official Tokio site?

**Comment by Noah-Kennedy** [comment]
I'm pretty sure its not, but @carllerche could you confirm?

**Comment by ewang2002** [comment]
https://tokio.uk.ms/ now redirects to https://tokio.rs/, so I suppose that answers the question.

**Reply by Darksonn** [reply, marked as answer]
The domain `tokio.uk.ms` is _not_ owned by the Tokio project.
